frontend/js/tcf-cmp.js

```
@@ -516,27 +607,29 @@
-				consent  = readConsent();
-				purposes = buildPurposeConsent(consent);
-				var sfAdd = buildSpecialFeatureOptins(consent);
-				tcStr    = encodeTcString(purposes, sfAdd);
-				data     = buildTCData(purposes, sfAdd, tcStr, listenerId);
-				data.eventStatus = "tcloaded";
-				callback(data, true);
-				break;
+			case "getTCData":
+				consent  = readConsent();
+				purposes = buildPurposeConsent(consent);
+				var sfGet = buildSpecialFeatureOptins(consent);
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)